diff -Naur QuiteRSS-0.17.0-src.orig/QuiteRSS.pro QuiteRSS-0.17.0-src/QuiteRSS.pro
--- QuiteRSS-0.17.0-src.orig/QuiteRSS.pro	2015-01-12 22:32:40.000000000 +0400
+++ QuiteRSS-0.17.0-src/QuiteRSS.pro	2015-01-12 22:33:05.000000000 +0400
@@ -278,7 +278,7 @@
   TARGET = quiterss
 
   isEmpty(PREFIX) {
-    PREFIX =   /usr/local
+    PREFIX =   /usr
   }
   DATA_DIR = $$PREFIX/share/quiterss
   DEFINES += RESOURCES_DIR='\\\"$${DATA_DIR}\\\"'
