--- a/src/application/mainwindow.cpp
+++ b/src/application/mainwindow.cpp
@@ -3335,7 +3335,7 @@
   bool showCloseButtonTab = settings.value("Settings/showCloseButtonTab", true).toBool();
   optionsDialog_->showCloseButtonTab_->setChecked(showCloseButtonTab);
 
-  bool updateCheckEnabled = settings.value("Settings/updateCheckEnabled", true).toBool();
+  bool updateCheckEnabled = false; //settings.value("Settings/updateCheckEnabled", true).toBool();
   optionsDialog_->updateCheckEnabled_->setChecked(updateCheckEnabled);
   bool statisticsEnabled = settings.value("Settings/statisticsEnabled2", true).toBool();
   optionsDialog_->statisticsEnabled_->setChecked(statisticsEnabled);
--- a/src/optionsdialog.cpp
+++ b/src/optionsdialog.cpp
@@ -298,6 +298,8 @@
   showCloseButtonTab_ = new QCheckBox(tr("Show close button on tab"));
 
   updateCheckEnabled_ = new QCheckBox(tr("Automatically check for updates"));
+  updateCheckEnabled_->setEnasabled(false);
+  updateCheckEnabled_->setCheckable(false);
   statisticsEnabled_ = new QCheckBox(tr("Help improve QuiteRSS by sending usage information"));
   storeDBMemory_ = new QCheckBox(tr("Store a DB in memory (requires program restart)"));
   storeDBMemory_->setChecked(false);
--- a/src/updateappdialog.cpp
+++ b/src/updateappdialog.cpp
@@ -218,8 +218,9 @@
 
 void UpdateAppDialog::renderStatistics()
 {
-  Settings settings;
-  bool updateCheckEnabled = settings.value("Settings/updateCheckEnabled", true).toBool();
+//  Settings settings;
+  bool updateCheckEnabled = false; //settings.value("Settings/updateCheckEnabled", true).toBool();
+/*
   if (updateCheckEnabled || showDialog_) {
     QNetworkRequest request(QUrl("http://quiterss.org/files/updates_new/VersionNo.h"));
     reply_ = networkManagerProxy_->get(request);
@@ -227,4 +228,5 @@
   } else {
     emit signalNewVersion();
   }
+*/
 }
